data/css/modules/layout/_sidebar.scss

```diff
@@ -7,7 +7,8 @@
         .CardTitle {
             margin: 0px;
             padding: 10px;
-            border-bottom: 2px solid transparentize(black, 1 - 0.1);
+            border-bottom: 1px solid transparentize(black, 1 - 0.2);
+            transition: all 0.2s ease;
```

Comment:
I think some changes don't corresponding to this commit? Please double check! :)